packages/cli-e2e/__tests__/react.specs.ts

```diff
@@ -229,17 +224,17 @@ describe('ui:create:react', () => {
       });
     });
 
-    it('should be commited without lint-stage errors', async () => {
-      const eslintErrorSpy = jest.fn();
+    it('should have a clean working directory', async () => {
+      const gitDirtyWorkingTreeSpy = jest.fn();
 
-      commitProject(
```

Comment:
There is no need to manually commit the /server directory since it is being done in the project creation.